community_images/metrics-server/ironbank/image.yml

```diff
@@ -0,0 +1,40 @@
+name: metrics-server
+official_name: Metrics-Server IronBank
+official_website: https://kubernetes.io/docs/tasks/debug/debug-cluster/resource-metrics-pipeline/
+source_image_provider: Platform One
+source_image_repo: registry1.dso.mil/ironbank/opensource/kubernetes-sigs/metrics-server
+source_image_repo_link: https://registry1.dso.mil/harbor/projects/3/repositories/opensource%2Fkubernetes-sigs%2Fmetrics-server
+source_image_readme: https://repo1.dso.mil/dsop/opensource/kubernetes-sigs/metrics-server/-/blob/development/README.md
+rf_docker_link: rapidfort/metrics-server-ib
+image_workflow_name: metrics-server_ironbank
+github_location: metrics-server/ironbank
+report_url: https://us01.rapidfort.com/app/community/imageinfo/registry1.dso.mil%2Fironbank%2Fopensource%2Fkubernetes-sigs%2Fmetrics-server
+usage_instructions: |
+  $ docker run rapidfort/metrics-server-ib
```

Comment:
Can we change usage instructions to helm command?

Reply:
Updated Usage Instructions